src/Components/src/Microsoft.AspNetCore.Components/CascadingValue.cs

```diff
@@ -49,13 +51,13 @@ public class CascadingValue<T> : ICascadingValueComponent, IComponent
         bool ICascadingValueComponent.CurrentValueIsFixed => IsFixed;
 
         /// <inheritdoc />
-        public void Init(RenderHandle renderHandle)
+        public void Configure(RenderHandle renderHandle)
         {
             _renderHandle = renderHandle;
         }
 
         /// <inheritdoc />
-        public void SetParameters(ParameterCollection parameters)
+        public async Task SetParametersAsync(ParameterCollection parameters)
```

Comment:
As per the comment below, I'm hoping this could remain a `void` method and we could avoid the `await` here and especially avoid the `Task.WhenAll` below. Will that still work the same? I would have thought so but let me know if I'm missing something.

Reply:
This implements IComponent, but I'll see if I can change it to simply do Task.CompletedTask and do fire and forget for the other things

Reply:
Ah, that sounds right!

Reply:
Done